README.md

```diff
@@ -15,45 +15,95 @@ Requires Rust 1.51.
 
 Create a new temporary project:
 
-* With no additional dependencies:
-    `$ cargo-temp`
-    
-* With multiple dependencies:
-    `$ cargo-temp rand tokio`
-    
-* When specifying a version:
-    `$ cargo-temp anyhow=1.0`
-    * Using the [cargo's comparison requirements][comparison]:
-        `$ cargo-temp anyhow==1.0.13`
+### With no additional dependencies:
+
+```bash
+cargo-temp
+```
+
+### With multiple dependencies:
+
+```bash
+cargo-temp rand tokio
+```
+
+### When specifying a version:
+
+```bash
+cargo-temp anyhow=1.0`
+```
+
+Using the [cargo's comparison requirements][comparison]:
+
+* Exact version:
+  ```bash
+  cargo-temp anyhow==1.0.13
+  ```
+
+* Maximal version:
+  ```bash
+  cargo-temp anyhow=<1.0.2
+  ```
```

Comment:
Pretty sure it was more readable with the bullets points instead of the titles :grin: it's like the 3rd time you or I attempt to titlize those bullet points and it's a terrible idea. Look at the preview